README.md

```diff
@@ -37,9 +37,11 @@ Installation
 DLite is available on PyPI and can be installed with pip
 
 ```shell
-pip install dlite-python
+pip install dlite-python[full]
 ```
 
+The bracket `[full]` is optional, but ensures that you install all optional
+dependencies together with DLite.
```

Comment:
Perhaps it is better to "flip this around", by which I mean: don't write `[full]` into the instructions here, but rather add it in a "Note" section below.

Reply:
From the comments of Anders and Sylvain, we should make it as easy as possible to install the full version of DLite as possible.

If we are not able to create a `dlite-python-minimal` package, I think the least we can do is to document how to retain the old behaviour when installing DLite.

Reply:
Okay. I guess that makes sense. Would it make sense to spell out a bit more (slightly) what it would mean to _not_ include the `[full]`? Maybe referring back up to the list of extra, optional functionalities that are not available?

Reply:
Ah - this line has already been added below this. Great.

Reply:
done